OpenRA.Game/HotkeyDefinition.cs

```diff
@@ -20,7 +20,7 @@ public sealed class HotkeyDefinition
 		public readonly Hotkey Default = Hotkey.Invalid;
 		public readonly string Description = "";
 		public readonly HashSet<string> Types = new HashSet<string>();
-		public bool HasDuplicates = false;
+		public bool HasDuplicates;
```

Comment:
Can you please also move the assignment from `SettingsLogic.SaveHotkey` to `HotkeyManager.Set` then make this `{ get; internal set; }`?

Reply:
Done.